modules/ztp-install-factory-pipeline.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="run-the-hub-cluster-factory-install-pipeline_{context}"]
+= Run the hub cluster factory install pipeline
+
+Follow the guidance here to run the hub factory install pipeline.
```

Comment:
Not a fan of "here". Suggest using the same wording you use elsewhere as the intro